engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/SchemaRegistryClient.scala

```diff
@@ -0,0 +1,25 @@
+package pl.touk.nussknacker.engine.avro
+
+import io.confluent.kafka.schemaregistry.client.SchemaMetadata
```

Comment:
woops it is isn't generic schema registry client :)

Reply:
You are right.. ;)